staging/kos/pkg/cmd/server/start.go

```diff
@@ -81,6 +86,7 @@ func NewCommandStartNetworkAPIServer(defaults *NetworkAPIServerOptions, stopCh <
 	flags := cmd.Flags()
 	o.RecommendedOptions.AddFlags(flags)
 	o.ServerRunOptions.AddUniversalFlags(flags)
+	flags.BoolVar(&o.CheckSubnetsConflicts, "check-subnets-conflicts", true, "Whether subnet creation/update fails if conflicting subnets are found. Setting it to false can ease testing of other control plane components.")
```

Comment:
I would be more specifc: "Setting it to false can ease testing of the subnet validator"; it is not clear to me that setting it to false helps testing of other control plane components.